ui/src/routes/_layout/organizations/$orgId/products/$productId/repositories/$repoId/_layout/runs/$runIndex/vulnerabilities/index.tsx

```diff
@@ -28,16 +45,148 @@ import {
   CardTitle,
 } from '@/components/ui/card';
 import { Label } from '@/components/ui/label';
+import { calcOverallVulnerability } from '@/helpers/calc-overall-vulnerability';
+import { getVulnerabilityRatingBackgroundColor } from '@/helpers/get-status-colors';
+import { paginationSchema } from '@/schemas';
+
+const defaultPageSize = 10;
+
+const columns: ColumnDef<VulnerabilityWithIdentifier>[] = [
+  {
+    accessorKey: 'vulnerability',
+    header: () => <></>,
+    cell: ({ row }) => {
+      const type = row.original.identifier.type;
+      const namespace = row.original.identifier.namespace;
+      const name = row.original.identifier.name;
+      const version = row.original.identifier.version;
```

Comment:
No need to change, but this could be more readable with object destructuring like `const { type, namespace, name, version } = row.original.identifier`.